backend/amd64/emit.mlp

```diff
@@ -1369,6 +1389,7 @@ let fundecl fundecl =
   else
     D.global (emit_symbol fundecl.fun_name);
   D.label (emit_symbol fundecl.fun_name);
+  D.label (label_name (emit_symbol fundecl.fun_name));
```

Comment:
Ah, this is something subtle that really does need a comment!

I think it is probably useful to unconditionally have symbols for functions, because they show up in gdb, perf, addr2line and other tools. So this code always outputs a real symbol for each function definition.

Reply:
Done